hasura/metadata/databases/default/tables/public_app.yaml

```diff
@@ -72,55 +62,31 @@ select_permissions:
       columns:
         - id
         - is_staging
-        - name
         - team_id
-      computed_fields:
-        - is_verified
-        - verified_app_logo
```

Comment:
you also need to drop these computed fields with a migration

Reply:
Dropped the functions but the fields are just removed in the metadata